test/CMakeLists.txt

```diff
@@ -1,7 +1,43 @@
 # (C) Copyright 2024 NOAA/NCEP/EMC.
-#
-# This software is licensed under the terms of the Apache Licence Version 2.0
-# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+# Download the test data if we don't have it.
+# Code borrowed from NCEPLIB-bufr project...
+
+#set(DOWNLOAD_URL "https://ftp.emc.ncep.noaa.gov/static_files/public")
+set(DOWNLOAD_URL "https://rmclaren.opalstacked.com/public")
```

Comment:
I hope you are not being billed for this.
@CoryMartin-NOAA FYI

Reply:
Can I provide you a location on the S3 bucket?  You will need to coordinate with the maintainer of the bucket, but at least all our test data will be in one place and on an authorized location.  
Do however note, that none of the test data should be restricted if being posted on a publicly accessed site.

Reply:
@aerorahul This is a server I maintain for my own purposes (have for a long time :)). I've got plenty of spare bandwidth available so it doesn't really cost me anything.

If anyone has access to the https://ftp.emc.ncep.noaa.gov/static_files/public site you could copy the the tgz file there, and I could change the URL.

Reply:
@aerorahul Feel free to share the S3 bucket location if you feel we should move the data there.

Reply:
I have to say, I don't think using a personal server is appropriate here. 
The FTP site is hosted on EMC rzdm.  I can start an email thread to get you access to EMC rzdm so you can manage the data independently.

Reply:
I've already talked to Daryl about getting access rzdm. Its a process...

Reply:
Sounds good.  Thanks for letting me know.

Reply:
Removed the URL